.travis.yml

```diff
@@ -17,11 +18,12 @@ cache:
     - $HOME/.composer/cache
 
 before_script:
-  - travis_retry composer install -n
+  - travis_retry composer self-update
```

Comment:
Why self-update?

Reply:
To use actual version of composer

Reply:
Why is it necessary? I'm sorry but if there's no reason to waste time on that then we shouldn't do it.